cmd/jujud/agent/model/manifolds.go

```diff
@@ -145,6 +145,7 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 			APIOpen:       api.Open,
 			NewConnection: apicaller.OnlyConnect,
 			Filter:        apiConnectFilter,
+			Logger:        config.LoggingContext.GetLogger("juju.worker.apicaller"),
```

Comment:
Right, and I guess this is the key part that gets the model-specific logger for this case.

Reply:
Yup